lnrpc/routerrpc/router_server_test.go

```diff
@@ -0,0 +1,169 @@
+package routerrpc
```

Comment:
Dope! We usually test RPC changes in itest, while itself is missing unit tests, so this is great!

Reply:
Yeah it's a bit easier to get the backend to give you what you want to test here. Otherwise you're dependent on pathfinding for example to test this method, which makes it a bit complicated.

Reply:
Exactly! I really like that we have this controlled environment to check the expected state changes in our system.